pos_remove_pos_category/static/src/js/pos_remove_pos_category.js

```diff
@@ -0,0 +1,37 @@
+/******************************************************************************
+ * Copyright (C) 2015-2016 Akretion (<http://www.akretion.com>).
+ * Copyright (C) 2017-TODAY Camptocamp SA (<http://www.camptocamp.com>).
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ ******************************************************************************/
+
+odoo.define('pos_remove_pos_category.remove_pos_category', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var pos_models = require('point_of_sale.models');
+    var pos_screens = require('point_of_sale.screens');
+    var _t = core._t;
```

Comment:
I think _t is not used. We can remove this.

Reply:
done